daemons/integration-test/app.py

```diff
@@ -0,0 +1,35 @@
+import os
+
+import domovoi
+import requests
+
+
+app = domovoi.Domovoi()
+
+
+@app.scheduled_function("rate(60 minutes)")
+def integration_test(*_):
+    travis_token = os.environ["TRAVIS_TOKEN"]
+    body = {
+        'request': {
+            'branch': "master",
+            'message': "INTEGRATION TESTS",
+            'config': {
+                'merge_mode': "deep_merge",
+                'env': {
+                    'matrix': ["TRAVIS_DSS_INTEGRATION_MODE=1"]
+                },
+            },
+        }}
+    headers = {
+        'Content-Type': "application/json",
```

Comment:
this header comes for free with `request(json=...)`

Reply:
Removed.